### DISCUSSION TITLE
Documentation of Vici

### DISCUSSION THREAD

**Discussion by fractalb** [strongswan/strongswan#981] [General]
I couldn't find `list-pools` command in the vici README file (https://github.com/strongswan/strongswan/blob/master/src/libcharon/plugins/vici/README.md). Is there some other documentation of vici which documents all the commands/events?

**Comment by tobiasbrunner** [comment]
`--list-pools` is a `swanctl` option, the vici command on which it is based is called [`get-pools()`](https://github.com/strongswan/strongswan/blob/master/src/libcharon/plugins/vici/README.md#get-pools).

**Reply by fractalb** [reply]
Thanks @tobiasbrunner for giving the exact command. I observed that the commands like `list-certs`, `list-sas`, `list-conns`, etc. of the `swanctl` correspond to the similarly named `vici` commands.  So, I assumed there must be a `list-pools` `vici` commands when I saw the `swanctl --list-pools` command. It was my bad. Finally, can I assume that the list of `vici` commands in the README.md is comprehensive? Or do I need to look in the source code to confirm that? Please advise.

**Comment by tobiasbrunner** [comment]
> Finally, can I assume that the list of vici commands in the README.md is comprehensive?

Probably, I've not checked in a while.
